R/drive_mkdir.R

```diff
@@ -54,9 +55,18 @@ drive_mkdir <- function(path = NULL, name = NULL, verbose = TRUE) {
 
   path <- path %||% root_folder()
   path <- as_dribble(path)
-  confirm_single_file(path)
+  if (!single_file(path)) {
```

Comment:
I was thinking in this case it couldn't contain 0 files since we have this `path <- path %||% root_folder()`, so if it were empty, it would default to the root folder.